In [6]:
Use TestMD

Commands completed successfully.

Total execution time: 00:00:00.012

In [7]:
SELECT * FROM TestForMD

(3 rows affected)

Total execution time: 00:00:00.042

ID Name Age Salary Height MaritalStatus 1 Tim 31 300.0000 191.20 1 2 Tom 21 400.0000 181.87 2 3 Tam 51 500.0000 176.54 3

##Result for table: _**TestForMD**_             
###SchemaName: _dbo_ 

|**ID**|**Name**|**Age**|**Salary**|**Height**|**MaritalStatus**|
|---|---|---|---|---|---| 
|1|Tim|31|300.00|191.20|1|
|2|Tom|21|400.00|181.87|2|  
|3|Tam|51|500.00|176.54|3| 




|---|---|---|---|---|---|

|1|Tim|31|300.00|191.20|1|

|2|Tom|21|400.00|181.87|2|

|3|Tam|51|500.00|176.54|3|